packages/rest/src/rest.server.ts

```diff
@@ -151,7 +151,20 @@ export class RestServer extends Context implements Server, HttpServerLike {
     return this._httpServer ? this._httpServer.listening : false;
   }
 
+  /**
+   * The base url for the server, including the basePath if set
+   */
   get url(): string | undefined {
+    let serverUrl = this.rootUrl;
+    if (!serverUrl) return serverUrl;
+    serverUrl = serverUrl + (this._basePath || '');
+    return serverUrl;
+  }
+
+  /**
+   * The root url for the server without the basePath
```

Comment:
Let's add an example value please. E.g. `http://127.0.0.1:3000/`Or is it `http://127.0.0.1:3000` (no trailing slash)? Please check and use the actual value.